dask/tests/test_base.py

```diff
@@ -627,3 +627,8 @@ def my_get(dsk, keys):
 
     with dask.set_options(array_optimize=None, get=my_get):
         y.compute()
+
+
+def test_class_is_not_collection():
+    da = pytest.importorskip('dask.array')
+    assert not is_dask_collection(da.Array)
```

Comment:
Would prefer to just add this to the bottom of `test_is_dask_collection` above:

```
assert not is_dask_collection(DummyCollection)
```

Reply:
Fixed